tools/nx-plugins/npm-package/.eslintrc.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
 			},
 			rules: {
 				'import/no-default-export': 'off', // Nx plugins export defaults
+				'@typescript-eslint/unbound-method': 'off',
```

Comment:
👀

Reply:
Was wondering if you'd spot that

Reply:
is this a new recommendation?

Reply:
not sure, but it wasn't flagging with the previous version of typescript. _think_ we're ok to ignore: https://typescript-eslint.io/rules/unbound-method/